### DISCUSSION TITLE
O Curso Full Cycle vale a pena?

### DISCUSSION THREAD

**Discussion by gabrielbugarelli** [frontendbr/forum#2306] [Discussão]
Fala pessoal, sou um dev fullstack javascript Júnior e gostaria de ampliar os meus conhecimentos em infra. Venho acompanhando as imersões e os conteúdos tem sido bem interessantes. Quero saber com vocês que  já fizeram o curso, como ele é na prática, vale a pena o investimento?

**Comment by wesley1263** [comment]
Oi Gabriel, beleza?
Cara , sobre sua pergunta,  depende!
Se você é um fullstack mais front do que back, não recomendo.
Pois as preocupações de um dev front, são diferentes de um dev back.
Para um front eu indicaria o curso da Rocketset ( acho que é assim que escreve).
Caso contrário,  super recomendo o curso.
